rbac/management/group/view.py

```diff
@@ -1083,10 +1132,23 @@ def roles(self, request, uuid=None, principals=None):
             serializer = GroupRoleSerializerIn(data=request.data)
             if serializer.is_valid(raise_exception=True):
                 roles = request.data.pop(ROLES_KEY, [])
+
             with transaction.atomic():
                 group = set_system_flag_before_update(group, request.tenant, request.user)
                 add_roles(group, roles, request.tenant, user=request.user)
+
             response_data = GroupRoleSerializerIn(group)
+            response = Response(status=status.HTTP_200_OK, data=response_data.data)
+            if status.is_success(response.status_code):
+                auditlog = AuditLog()
+                auditlog.log_group_assignment(
+                    request,
+                    AuditLog.GROUP,
+                    object=group,
+                    type_dict=response_data.data,
```

Comment:
what about to pass just values for description ? Something like:
`[response_data.data["uuid"] for role in response_data.data]`

then type_dict could be renamed to something like `values_for_description`
and it will allow to remove method `self.find_specific_list_of_users` and it is possible to join(",") in log assignment method.